databuilder/transformer/table_tag_transformer.py

```diff
@@ -0,0 +1,31 @@
+from pyhocon import ConfigFactory
+
+from databuilder.transformer.base_transformer import Transformer
+from databuilder.models.table_metadata import TableMetadata
+
+
+class TableTagTransformer(Transformer):
+    """Simple transformer that adds tags to all table nodes produced as part of a job."""
+    # Config
+    TAGS = "tags"
+    DEFAULT_CONFIG = ConfigFactory.from_dict({TAGS: None})
+
+    def init(self, conf):
+        conf = conf.with_fallback(TableTagTransformer.DEFAULT_CONFIG)
+        tags = conf.get_string(TableTagTransformer.TAGS)
+
+        if isinstance(tags, str):
+            tags = tags.split(",")
+        self.tags = tags
+
+    def transform(self, record):
+        if isinstance(record, TableMetadata):
+            if record.tags:
+                record.tags += self.tags
```

Comment:
I think tags could be string or list based on https://github.com/lyft/amundsendatabuilder/blob/master/databuilder/models/table_metadata.py#L238, so purely do `+=` may not work for both cases.